src/components/DropdownHeader/index.js

```diff
@@ -5,7 +5,7 @@ import cn from 'classnames'
 import { PROP_TYPE_CHILDREN } from '../constants'
 
 
-export default class DropdownFooter extends PureComponent {
+export default class DropdownHeader extends PureComponent {
```

Comment:
Not sure how this change got squeezed into this branch, but I can remove if desired

Reply:
how dare you name it what it should be named.  and who is the fool that named this incorrectly in the first place???

oh, it me...